diablo/jobs/instructor_emails_job.py

```diff
@@ -89,9 +93,10 @@ def email_scheduled_courses(self):
                                 subject_line=subject_line,
                                 recipient=instructor,
                                 section_id=course['sectionId'],
-                                template_type='invitation',
+                                template_type=template_type,
```

Comment:
I discovered this copy-and-paste blunder. The `queued_emails` table was getting wrong `template_type` which apparently has been benign.